macros/sql/deduplicate.sql

```diff
@@ -29,10 +29,17 @@
 
 {%- endmacro -%}
 
-{# Redshift should use default instead of Postgres #}
+-- Redshift has the `QUALIFY` syntax:
+-- https://docs.aws.amazon.com/redshift/latest/dg/r_QUALIFY_clause.html
 {% macro redshift__deduplicate(relation, partition_by, order_by) -%}
 
-    {{ return(dbt_utils.default__deduplicate(relation, partition_by, order_by=order_by)) }}
+    select *
+    from {{ relation }} as tt
```

Comment:
@yauhen-sobaleu Does it still work if we peel out the alias like this?
```suggestion
    from {{ relation }}
```

If so, it would be nice to use the same exact implementation as databricks and snowflake.

Reply:
Hello @dbeatty10 . Surprisingly it does not work without an alias. Usage of alias is intended.

```
ERROR: syntax error at or near "row_number" Position: 312 [ErrorId: 1-64b8e1ad-0f78fd2e0ec83de90652f133]
```